README.md

```diff
@@ -11,7 +11,7 @@ http://www.python.org/dev/peps/pep-0381/.
 The following instructions will place the bandersnatch executable in a
 virtualenv under `bandersnatch/bin/bandersnatch`.
 
-- bandersnatch **requires** `>= Python 3.5`
+- bandersnatch **requires** `>= Python 3.6`
```

Comment:
I certainly would love to require Python 3.6+ however I suspect there are a number of users of bandersnatch that are using operating systems that don't provide python releases newer than 3.5.  In which case requiring 3.6 would be a breaking change.  It seems like at minimum we need to up the major number if in fact we are dropping support for 3.5.

Reply:
Yeah I was torn here, but aiohttp made the call to be >= 3.6 and I want to move there ... We can still sync in 3.5 but not verify due to aiohttp.